packages/pluggableWidgets/maps-native/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
     "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
     "baseUrl": "./",
-    "include": ["./src", "./typings"]
+    "include": ["./src", "./typings", "node_modules/@mendix/pluggable-widgets-tools/typings"]
```

Comment:
Can you remind me why we are adding this again ?

Reply:
for the SVG typings, all though it should only be in the preview branch. But I guess no problem to already include it in this one too